packages/grid/x-grid/src/tests/keyboard.XGrid.test.tsx

```diff
@@ -2,11 +2,11 @@ import { XGrid } from '@material-ui/x-grid/XGrid';
 import { waitFor } from '@testing-library/react';
 import { expect } from 'chai';
 import * as React from 'react';
-import { getActiveCell } from '../../../../test/utils/helperFn';
+import { getActiveCell } from '../../../../../test/utils/helperFn';
```

Comment:
```suggestion
import { getActiveCell } from 'test/utils/helperFn';
```

x all the other places